content/docs/data-api/get-started.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Getting started with Neon Data API
+description: Learn how to use the Neon Data API, a ready-to-use REST API built on top of your Neon database
+enableTableOfContents: true
+---
+
+<PrivatePreview />
+
+<InfoBlock>
+  <DocsList title="Related docs" theme="docs">
+    <a href="/docs/guides/neon-auth">Neon Auth</a>
+  </DocsList>
+  <DocsList title="Demo app" theme="repo">
+    <a href="https://github.com/neondatabase-labs/neon-data-api-neon-auth">Neon Data API Demo App</a>
+  </DocsList>
+</InfoBlock>
+
+The Neon Data API is a ready-to-use REST API for your Neon database, powered by [PostgREST](https://docs.postgrest.org/en/v13/), a trusted project in the PostgreSQL community. It lets you work with every table, view, or function in a database's schema using standard HTTP verbs. More importantly, this means you can use a nice SDK like `Postgrest.js` to run queries from the client, like this:
```

Comment:
`you can use a nice SDK like Postgrest.js, or Postgrest.py, or Postgrest-go to run queries from the client`

I think it'd be nice to mention the SDKs for multiple languages